Tests/Common/TestUtils.cs

```diff
@@ -28,7 +28,7 @@ public static class TestUtils
             ILoRaDeviceClientConnectionManager connectionManager,
             DefaultLoRaDataRequestHandler requestHandler = null)
         {
-            var result = new LoRaDevice(simulatedDevice.LoRaDevice.DevAddr, simulatedDevice.LoRaDevice.DeviceID, connectionManager)
+            var result = new LoRaDevice(simulatedDevice.LoRaDevice.DevAddr, DevEui.Parse(simulatedDevice.LoRaDevice.DeviceID), connectionManager)
```

Comment:
Use `DevEui` already added to `LoRaDevice`:

```suggestion
            var result = new LoRaDevice(simulatedDevice.LoRaDevice.DevAddr, simulatedDevice.LoRaDevice.DevEui, connectionManager)
```